src/test/javascript/shared/utils.ts

```diff
@@ -0,0 +1,13 @@
+export const createMutationOnCurationPage = async (mutationName: string) => {
+  const mutationNameInput = await $('input#add-mutation-modal-input');
+  await mutationNameInput.setValue(mutationName);
```

Comment:
When adding mutation, we trigger an api call to annotate. Do we mock the data currently? Are  different mutation names using the same mock?

Reply:
Yeah they will all use the same mock since the API endpoint to annotate is the same (mutation name is passed in the request body). It may be possible to mock based on request body.